diff --git a/lib/cli.js b/lib/cli.js
index 06c295f123c479dbd7b6b2ca5b842cdf81a2dfff..7ba6a3a6331f2d391565da0d2be24a55c79a2c21 100644
--- a/lib/cli.js
+++ b/lib/cli.js
@@ -1,8 +1,7 @@
 import { Command } from 'commander';
 import { logo } from './utils/logo.js';
-import { createRequire } from 'module';
-const require = createRequire(import.meta.url);
-const { version } = require('../package.json');
+import pkg from '../package.json' with { type: 'json' }; 
+const { version } = pkg;
 const program = new Command();
 
 // Create Gasket CLI
diff --git a/lib/create.js b/lib/create.js
index f456475b554ca8dc64a8810369d9607c04c20887..5139a2e2cb6e38d5a8538392506f5493d6ed508e 100644
--- a/lib/create.js
+++ b/lib/create.js
@@ -1,8 +1,7 @@
 /// <reference types="create-gasket-app" />
 
-import { createRequire } from 'module';
-const require = createRequire(import.meta.url);
-const { name, version } = require('../package.json');
+import pkg from '../package.json' with { type: 'json' }; 
+const { name, version } = pkg;
 
 /** @type {import('@gasket/core').HookHandler<'create'>} */
 export default function create(gasket, { pkg, gasketConfig }) {
diff --git a/lib/index.js b/lib/index.js
index b2059d13ef0519651345a6eb96946c9e1b0ad0ce..2f6fa246fa691e2187b63e3062ef166118542a05 100644
--- a/lib/index.js
+++ b/lib/index.js
@@ -5,9 +5,8 @@ import configure from './configure.js';
 import prepare from './prepare.js';
 import commands from './commands.js';
 import ready from './ready.js';
-import { createRequire } from 'module';
-const require = createRequire(import.meta.url);
-const { name, version, description } = require('../package.json');
+import pkg from '../package.json' with { type: 'json' }; 
+const { name, version, description } = pkg;
 
 /** @type {import('@gasket/core').Plugin} */
 export default {
